docs/tutorial/writing-documentation-with-docz.md

```diff
@@ -27,16 +27,18 @@ cd my-gatsby-site-with-docz
 And install the following packages:
 
 ```shell
-npm install --save gatsby-theme-docz docz docz-theme-default
+npm install --save gatsby-theme-docz
```

Comment:
i'm not 100% sure but this is what's on the [docz page](https://www.docz.site/docs/gatsby-theme)
`npm install --save gatsby-theme-docz@next docz@next`

Maybe ping @pedronauck

Reply:
Yeah, I checked. Theme README has what I added so we're good! `docz` is already a dependency of the theme itself

Reply:
@sidharthachatterjee   i  think this still needs the @next version specifier since gatsby-theme-docz 2.0.0 is stlll rc.1 so npm install only installs version 1.2.0 that still requires docz-theme-default

Reply:
Just checked again and you’re absolutely right, @jstacoder 

Want to update this? 🙂

Reply:
@sidharthachatterjee sure ill make a PR to fix that

Reply:
Thanks a tonne @jstacoder 🙌🏼